src/components/dialogs/network-modifications/static-var-compensator/creation/standby-automaton-form-utils.ts

```diff
@@ -0,0 +1,121 @@
+import { computeQAtNominalV } from '../../../../utils/utils';
+
+export const getStandbyAutomatonEmptyFormData = (id = AUTOMATON) => ({
+    [id]: {
+        [ADD_STAND_BY_AUTOMATON]: false,
+        [CHARACTERISTICS_CHOICE_AUTOMATON]: CHARACTERISTICS_CHOICES.Q_AT_NOMINAL_V.id,
+        [STAND_BY_AUTOMATON]: false,
+        [LOW_VOLTAGE_SET_POINT]: null,
+        [HIGH_VOLTAGE_SET_POINT]: null,
+        [LOW_VOLTAGE_THRESHOLD]: null,
+        [HIGH_VOLTAGE_THRESHOLD]: null,
+        [SLIDER_SUSCEPTANCE]: null,
+        [SLIDER_Q_NOMINAL]: null,
+        [B0]: null,
+        [Q0]: null,
+    },
+});
+
+const requiredIfAddStandbyAutomaton = (yup: any) =>
+    yup.nullable().when([ADD_STAND_BY_AUTOMATON], {
+        is: true,
+        then: (schema: any) => schema.required(),
+    });
+
+const requiredIfAddStandbyAutomatonAndSusceptanceChoice = (yup: any) =>
+    yup.nullable().when([ADD_STAND_BY_AUTOMATON, CHARACTERISTICS_CHOICE_AUTOMATON], {
+        is: (addStandbyAutomaton: boolean, characteristicsChoiceAutomaton: string) =>
+            addStandbyAutomaton && characteristicsChoiceAutomaton === CHARACTERISTICS_CHOICES.SUSCEPTANCE.id,
+        then: (schema: any) => schema.required(),
+    });
+
+const requiredIfAddStandbyAutomatonAndQaTNominalVChoice = (yup: any) =>
+    yup.nullable().when([ADD_STAND_BY_AUTOMATON, CHARACTERISTICS_CHOICE_AUTOMATON], {
+        is: (addStandbyAutomaton: boolean, characteristicsChoiceAutomaton: string) =>
+            addStandbyAutomaton && characteristicsChoiceAutomaton === CHARACTERISTICS_CHOICES.Q_AT_NOMINAL_V.id,
+        then: (schema: any) => schema.required(),
+    });
+
+export const getStandbyAutomatonFormValidationSchema = () =>
+    yup.object().shape({
+        [ADD_STAND_BY_AUTOMATON]: yup.boolean().nullable(),
+        [STAND_BY_AUTOMATON]: yup
+            .boolean()
+            .nullable()
+            .when([ADD_STAND_BY_AUTOMATON, VOLTAGE_REGULATION_MODE], {
+                is: (addStandbyAutomaton: boolean, voltageRegulationMode: string) =>
+                    addStandbyAutomaton && voltageRegulationMode === VOLTAGE_REGULATION_MODES.VOLTAGE.id,
+                then: (schema) => schema.required(),
+            }),
+        [LOW_VOLTAGE_SET_POINT]: requiredIfAddStandbyAutomaton(yup.number()),
+        [HIGH_VOLTAGE_SET_POINT]: requiredIfAddStandbyAutomaton(yup.number()),
+        [LOW_VOLTAGE_THRESHOLD]: requiredIfAddStandbyAutomaton(yup.number()),
+        [HIGH_VOLTAGE_THRESHOLD]: requiredIfAddStandbyAutomaton(yup.number()),
+        [CHARACTERISTICS_CHOICE_AUTOMATON]: requiredIfAddStandbyAutomaton(yup.string()),
+        [B0]: requiredIfAddStandbyAutomatonAndSusceptanceChoice(yup.number()),
+        [Q0]: requiredIfAddStandbyAutomatonAndQaTNominalVChoice(yup.number()),
+        [SLIDER_SUSCEPTANCE]: requiredIfAddStandbyAutomatonAndSusceptanceChoice(yup.number()),
+        [SLIDER_Q_NOMINAL]: requiredIfAddStandbyAutomatonAndQaTNominalVChoice(yup.number()),
```

Comment:
We can remove these validations. When switching from Susceptance to Q at Nominal in tab 'Automaton', if min max of Q at Nominal are not set in tab 'Set Points & Limit', slider value will be set to null, even Fixed value is filled with a value, the form can not be validated and the slider can not be changed to red.